mappings/net/minecraft/recipe/RecipeManager.mapping

```diff
@@ -1,42 +1,126 @@
 CLASS net/minecraft/class_1863 net/minecraft/recipe/RecipeManager
+	COMMENT A recipe manager allows easier use of recipes, such as finding matches and
+	COMMENT remainders. It is also integrated with a recipe loader, which loads recipes
+	COMMENT from data packs' JSON files.
 	FIELD field_19359 GSON Lcom/google/gson/Gson;
 	FIELD field_9023 recipes Ljava/util/Map;
 	FIELD field_9024 errored Z
+		COMMENT This isn't quite indicating an errored state; its value is only set to
+		COMMENT {@code false} and is never {@code true}, and isn't used anywhere.
```

Comment:
Probably an old field that lost its uses along the way

Reply:
Very likely, but the current use imo is best dissociated from its past use, as otherwise it may be bug prone